bootstrap.sh

```diff
@@ -35,39 +35,47 @@ then
     then
         echo "Downloading clang-tidy and clang-format to bin directory"
```

Comment:
Do we control the directory structure of clang-tools?  If so, why not make the folders match the RID so you just use it to directly derive the path name.  Fail with a useful message if the dir doesn't exist.  Then, to add new platform support,  we just upload some files to the blob and the script doesn't need to change.

If we can't do that, we should refactor this code to be data-driven.  Create a table of pairs ("osx" -> "osx", "ubuntu.16.04" -> "ubuntu/16.04"...) that will let you derive the name.

At the very least, the common code in these blocks should be pulled out into a helper that takes platform ID and directory fragment as parameters.